Formula/fdclone.rb

```diff
@@ -1,8 +1,8 @@
 class Fdclone < Formula
   desc "Console-based file manager"
   homepage "https://hp.vector.co.jp/authors/VA012337/soft/fd/"
-  url "https://hp.vector.co.jp/authors/VA012337/soft/fd/FD-3.01b.tar.gz"
-  sha256 "d66d902cac9d4f64a91d42ceb487a138d544c9fd9cb2961730889cc8830303d4"
+  url "http://ftp.vector.co.jp/69/81/1362/FD-3.01d.tar.gz"
```

Comment:
What's with the protocol downgrade here?

Reply:
I think we'll have to ask upstream. Apparently, this isn't even the canonical URL, which is actually http://www.unixusers.net/src/fdclone/FD-3.01d.tar.gz